### PR TITLE
Default Admin Login - Apache Axis2

### PR DESCRIPTION
**Pull request by Prince-Mendiratta**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,10 +5,9 @@ All notable changes to this application will be documented in this file.
...
 to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## Unreleased
+## [Unreleased]
 
-### Added
-- Vulnerability: Host Header Injection
-- sitemap.xml
-- Dockerfile and docker-compose.yml
-- Procfile for deploying to heroku
```
Comment (ricekot) [inline CHANGELOG.md:14]: We aren't creating a release right now, so let's just keep everything under Unreleased. Also can you add all the vulnerabilities to the list?

File: src/app/vulnerabilities/Apache Axis2 Default Login.php [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+);
+
+$app->post(
+    '/axis2-admin/login',
+    function (Request $request, Response $response, array $args) {
+        $_body = $request->getParsedBody();
+        $_usname = $_body['userName'];
+        $_pass = $_body['password'];
+        
+        if(!empty($_usname && !empty($_pass))){
```
Comment (ricekot) [inline src/app/vulnerabilities/Apache Axis2 Default Login.php:34]:
```suggestion
        if(!empty($_usname) && !empty($_pass)){
```

File: src/app/resources/axis2/welcome.html [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (ricekot) [inline src/app/resources/axis2/welcome.html:80]: why all these empty lines?

File: src/app/resources/axis2/welcome.html [hunk 4]
```diff
@@ -0,0 +1,194 @@
...
+                </td>
+              </tr>
+            </tbody></table>
+          </form>
+          <br><br><br><br><br><br>
+        </td>
+      </tr>
+    </tbody></table>   
+    <!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one
```
Comment (ricekot) [inline src/app/resources/axis2/welcome.html:147]: Do we need the same license twice in the same file? Same comment for the other file.

**Review by ricekot — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (Prince-Mendiratta) [reply CHANGELOG.md]: Resolved.

File: src/app/vulnerabilities/Apache Axis2 Default Login.php (hunk 2 above)
Comment (Prince-Mendiratta) [reply src/app/vulnerabilities/Apache Axis2 Default Login.php]: Thanks!

File: src/app/resources/axis2/welcome.html (hunk 3 above)
Comment (Prince-Mendiratta) [reply src/app/resources/axis2/welcome.html]: The raw HTML file wasn't really good looking haha. Have fixed this.

File: src/app/resources/axis2/welcome.html (hunk 4 above)
Comment (Prince-Mendiratta) [reply src/app/resources/axis2/welcome.html]: Nope. A single license should be enough. Resolved.

File: CHANGELOG.md [hunk 5]
```diff
@@ -8,7 +8,17 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
...
+- Vulnerability: Host Header Injection
+- Vulnerability: Default Admin Login - Apache Axis2
+- Vulnerability: Apache Tomcat Ghostcat CVE 2020-1938
+- Vulnerability: Hidden File Sample
+- Vulnerability: JSP Samples Page
+- Vulnerability: Exposed Panels - CrushFTP
+
+### Changed
+- Restructure Project
+- GIF Favicon
```
Comment (ricekot) [inline CHANGELOG.md:23]: This was also added, not changed

Comment (Prince-Mendiratta) [reply CHANGELOG.md:23]: The `Restructure Project` would come under changed, right?

Comment (ricekot) [reply CHANGELOG.md:23]: yeah

Comment (Prince-Mendiratta) [reply CHANGELOG.md:21]: Changed.

**Pull request merged**